dashboard/app/controllers/api/v1/census/census_controller.rb

```diff
@@ -100,6 +102,13 @@ def new_submission
         school_info.save
         submission.save
       end
+      if template
+        recipient = Poste2.create_recipient(submission.submitter_email_address,
+          name: submission.submitter_name,
+          ip_address: request.remote_ip
+        )
+        Poste2.send_message(template, recipient)
+      end
```

Comment:
This LGTM, but I wonder if we're doing this on the dashboard side whether we should setup a [Rails mailer](http://guides.rubyonrails.org/action_mailer_basics.html) for it instead of going directly through Poste. Maybe something for the tech debt list. Our ActionMailer is setup to go through Poste anyway, but it would be nice if dashboard had no direct dependencies on it (in case we ever decide to decouple them).

Reply:
Ah, I didn't realize that there was a different way to handle emails in dashboard. Post Thanksgiving I will look into doing it that way. For now, I'm going to stick with this to make sure the fix gets in before lockdown.